docs/creating-a-new-adapter.md

```diff
@@ -0,0 +1,82 @@
+---
+layout: docs
+permalink: /docs/creating-a-new-adapter/index.html
+---
+
+# Adapter Basics
+
+All adapters inherit from the Adapter class in the `src/adapter.coffee` file.  There are certain methods that you will want to override.  Here is a basic stub of what an extended Adapter class would look like:
+
+```coffee
+class Bunny extends Adapter
+
+  constructor: ->
+    @robot.logger.info "Constructor"
+    super
+    
+
+  send: (envelope, strings...) ->
+    @robot.logger.info "Send"
+
+  reply: (envelope, strings...) ->
+    @robot.logger.info "Reply"
+
+  run: ->
```

Comment:
I think the only thing this is missing is actually constructing messages and passing them into `receive`.